### PR TITLE
Remove dedicated black/whitelist pages + redesign menu

### PR DESCRIPTION
**Pull request by yubiuser**
- **What does this PR aim to accomplish?:**

This PR removes the separate Black/Whitelist pages from the nav menu. They have been there to aid new users and to make the transition pre-v5 -> v5 more easy. It was always planed to remove them at some point.
They have been only a font-end for `groups-domais.php` where the `group` column was hidden.

The whole functionality can still be accessed via `Group Management` -> `Domains`. 

After removing the two entries the menu looked quite empty so I re-structured it a bit. Additional, added a link to our Discourse forum.

- **How does this PR accomplish the above?:**

Remove now unused code.

---
**By submitting this pull request, I confirm the following:** 

1. I have read and understood the [contributors guide](https://docs.pi-hole.net/guides/github/contributing/), as well as this entire template. I understand which branch to base my commits and Pull Requests against. 
2. I have commented my proposed changes within the code and I have tested my changes.
3. I am willing to help maintain this change if there are issues with it later.
4. It is compatible with the [EUPL 1.2 license](https://opensource.org/licenses/EUPL-1.1)
5. I have squashed any insignificant commits. ([`git rebase`](http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html))

---
- [x] I have read the above and my PR is ready for review. _Check this box to confirm_

### CODE REVIEW

**Comment by yubiuser** [comment]
Proposed new menu structure

Logged out:
Bildschirmfoto zu 2022-07-16 11-08-50

Logged in:
Bildschirmfoto zu 2022-07-16 11-08-26

Without password set:
Bildschirmfoto zu 2022-07-16 11-15-32

**Comment by DL6ER** [comment]
I propose removing "GROUP" from "GROUP MANAGEMENT", see what I added for branch `new/FTL_is_my_new_home`:

Screenshot from 2022-07-16 13-02-57

**Comment by yubiuser** [comment]
That's where I took it from but deliberately added the `Group` because all the sub menu items deal with the group management. Also `management` alone is very close to `System` or `Settings`

**Comment by yubiuser** [comment]
Ready for review. I guess there might be more code that could go...

**Comment by rdwebdesign** [comment]
There is something wrong with the "Collapsed" menu.

image

I can look into it later (or tomorrow).

File: scripts/pi-hole/php/header.php [hunk 1]
```diff
@@ -433,7 +445,7 @@ function pidofFTL()
                     </li>
                     <li<?php if($scriptname === "db_queries.php"){ ?> class="active"<?php } ?>>
                         <a href="db_queries.php">
-                            <i class="fa fa-fw menu-icon fa-file-alt"></i> Query Log
+                            <i class="fa fa-fwDNS Control menu-icon fa-file-alt"></i> Query Log
```
Comment (rdwebdesign) [inline scripts/pi-hole/php/header.php]: I'm not sure why you added "DNS Control" classes here, but i think **you forgot the space**:
```suggestion
                            <i class="fa fa-fw DNS Control menu-icon fa-file-alt"></i> Query Log
```

Or remove these extra classes, if it was a mistake:
```suggestion
                            <i class="fa fa-fw menu-icon fa-file-alt"></i> Query Log
```

File: style/themes/default-light.css [hunk 2]
```diff
@@ -64,7 +64,7 @@
   color: #fff;
 }
 .sidebar-menu > li.header {
-  color: #4b646f;
+  color: rgb(173, 166, 156);
```
Comment (rdwebdesign) [inline style/themes/default-light.css]: In my opinion, the color you chose (`rgb(173, 166, 156)`) is perfect for the Darker theme, but I think It is too "yellowish" to be used with other themes.

My suggestion to Light theme is:
`#808a90` (or `rgb(128, 138, 144)`, if you want to use rgb() function).

File: style/themes/default-dark.css [hunk 3]
```diff
@@ -92,7 +92,7 @@ h4 {
   color: #fff;
 }
 .sidebar-menu > li.header {
-  color: #556068;
+  color: rgb(173, 166, 156);
```
Comment (rdwebdesign) [inline style/themes/default-dark.css]: Same here. My suggestion to Dark theme:
`#8e989d` (or `rgb(143,152,158)`).

**Review by rdwebdesign — CHANGES_REQUESTED** [review]

File: scripts/pi-hole/php/header.php (hunk 1 above)
Comment (yubiuser) [reply scripts/pi-hole/php/header.php]: Clearly a copy&pasta in the wrong spot.

**Review by DL6ER — DISMISSED** [review]

**Review by DL6ER — APPROVED** [review]

**Review by rdwebdesign — APPROVED** [review]

**Pull request merged**